content/teaching-curriculum/project-quick_familiarity_1

```diff
@@ -0,0 +1,28 @@
+---
+_db_id: 
+content_type: project
+flavours:
+- teaching
+learning_outcomes: null
+prerequisites:
+  hard: []
+  soft: []
+ready: true
+story_points: null
+submission_type: repo
+tags: 
+title: Quick Familiarity - Part 1
+---
+
+
+As mentioned in the introductory topic of the course, the point of the quick familiarity projects is to do a quick skim of all of the techniques in the book, just to introduce you to the ideas and make sure that we can all start using the common language that the book provides for talking about nuanced teaching practices. We’ll practice the techniques more thoroughly later, but for now let's get just acquainted with them.
+
+[This 12 page book summary](https://www.boyd.k12.ky.us/userfiles/496/Classes/27400/Teach%20Like%20A%20Champion%20-%20The%20Main%20Idea.pdf) provides a really nice 1 page intro to the methodology that author Doug Lemov and team used to distil the best teaching practices from the best teachers in the country. 
+
+To keep the projects bite-sized, we'll look at just 3 chapters at a time and do a little bit of light reflection and planning. 
+
+## Your Mission Objectives
+* Read through the first 5 pages of the summary pdf (up to the end of chapter 3). 
+* For each of the 3 chapters: 
+    * (A) Describe which of the techniques you already apply pretty well, in general, and where and how it is that you tend do so. Also explain how it is that you know it works. 
+    * (B) Describe which of the techniques, if applied better or more consistently, you think has the greatest potential to improve your teaching practice. Be sure to motivate your choice with some sound reasoning.  
```

Comment:
Should they submit a markdown file? Please always be explicit about expectations